Scripts/Interfaces/IUsesToolTip.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace UnityEngine.Experimental.EditorVR
+{
+	/// <summary>
+	/// Provides access to the ability to show or hide a Tooltip
+	/// </summary>
+	public interface IUsesTooltip
```

Comment:
And it looks like it's necessary for anything that wants to show tooltips to use this method? No tooltips would show automatically?